toolset/setup/linux/installer.py

```diff
@@ -98,16 +98,6 @@ def __install_server_software(self):
       previousDir = os.getcwd()
       os.chdir(test_dir)
 
-      # Load profile for this installation
-      profile="%s/bash_profile.sh" % test_dir
-      if not os.path.exists(profile):
-        profile="$FWROOT/config/benchmark_profile"
-      else:
-        logging.info("Loading environment from %s (cwd=%s)", profile, test_dir)
-      setup_util.replace_environ(config=profile, 
```

Comment:
This is incorrect IMO. We still need to load the `benchmark_profile` file, so this should read: 

```
profile="$FWROOT/config/benchmark_profile"
setup_util.replace_environ(config=profile, 
  command='export TROOT=%s && export IROOT=%s' %
  (test_dir, test_install_dir))
```